sources/packages/web/src/components/common/OfferingForm.vue

```diff
@@ -96,15 +98,30 @@ export default defineComponent({
     },
   },
   setup(props, context) {
+    const { institutionState } = useInstitutionState();
```

Comment:
This is an institution-specific composable. Should the below variables not be populated from the consumer of this component instead?
`isInstitutionBCPublic: institutionState.value.isBCPublic,`
`isInstitutionBCPrivate: institutionState.value.isBCPrivate,`

For instance, if they are needed in the `OfferingCreate.vue` that is an institution-specific component, the `OfferingCreate.vue` can populate the initial data with these two properties using the `useInstitutionState` from the institution. Please let me know if I am missing something or if additional clarification is required.